<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="45" started-at="2017-03-15T10:37:26Z" finished-at="2017-03-15T10:37:26Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="45" started-at="2017-03-15T10:37:26Z" finished-at="2017-03-15T10:37:26Z">
      <class name="testscripts.ArticlePage.TC1">
        <test-method status="PASS" signature="suiteSetup(org.testng.ITestContext)[pri:0, instance:testscripts.ArticlePage.TC1@39aeed2f]" name="suiteSetup" is-config="true" duration-ms="622" started-at="2017-03-15T03:37:26Z" finished-at="2017-03-15T03:37:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@2b662a77]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- suiteSetup -->
        <test-method status="PASS" signature="testMethodSetup()[pri:0, instance:testscripts.ArticlePage.TC1@39aeed2f]" name="testMethodSetup" is-config="true" duration-ms="2" started-at="2017-03-15T03:37:26Z" finished-at="2017-03-15T03:37:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMethodSetup -->
        <test-method status="FAIL" signature="runTC1()[pri:0, instance:testscripts.ArticlePage.TC1@39aeed2f]" name="runTC1" duration-ms="14" started-at="2017-03-15T03:37:26Z" finished-at="2017-03-15T03:37:26Z">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[No enum constant com.cognizant.framework.selenium.Browser.ie]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: No enum constant com.cognizant.framework.selenium.Browser.ie
	at java.lang.Enum.valueOf(Enum.java:238)
	at com.cognizant.framework.selenium.Browser.valueOf(Browser.java:1)
	at supportlibraries.DriverScript.setDefaultTestParameters(DriverScript.java:125)
	at supportlibraries.DriverScript.startUp(DriverScript.java:112)
	at supportlibraries.DriverScript.driveTestExecution(DriverScript.java:93)
	at testscripts.ArticlePage.TC1.runTC1(TC1.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runTC1 -->
        <test-method status="FAIL" signature="testMethodTearDown()[pri:0, instance:testscripts.ArticlePage.TC1@39aeed2f]" name="testMethodTearDown" is-config="true" duration-ms="0" started-at="2017-03-15T03:37:26Z" finished-at="2017-03-15T03:37:26Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.cognizant.framework.Report.updateResultSummary(Report.java:381)
	at com.cognizant.framework.selenium.ResultSummaryManager.updateResultSummary(ResultSummaryManager.java:145)
	at supportlibraries.TestCase.testMethodTearDown(TestCase.java:93)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:712)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMethodTearDown -->
        <test-method status="SKIP" signature="suiteTearDown()[pri:0, instance:testscripts.ArticlePage.TC1@39aeed2f]" name="suiteTearDown" is-config="true" duration-ms="0" started-at="2017-03-15T03:37:26Z" finished-at="2017-03-15T03:37:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- suiteTearDown -->
      </class> <!-- testscripts.ArticlePage.TC1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
